src/unit-taxonomy-tags-drawer/TagBubble.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { render } from '@testing-library/react';
+import PropTypes from 'prop-types';
+
+import initializeStore from '../store';
+
+import TagBubble from './TagBubble';
+
+let store;
+
+const data = {
+  value: 'Tag 1',
+};
+
+const TagBubbleComponent = ({ value, subTagsCount, implicit }) => (
+  <AppProvider store={store}>
+    <IntlProvider locale="en" messages={{}}>
+      <TagBubble value={value} subTagsCount={subTagsCount} implicit={implicit} />
+    </IntlProvider>
+  </AppProvider>
+);
+
+TagBubbleComponent.defaultProps = {
+  subTagsCount: 0,
+  implicit: true,
+};
+
+TagBubbleComponent.propTypes = {
+  value: PropTypes.string.isRequired,
+  subTagsCount: PropTypes.number,
+  implicit: PropTypes.bool,
+};
+
+describe('<TagBubble />', async () => {
+  beforeEach(async () => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: true,
+        roles: [],
+      },
+    });
+    store = initializeStore();
+  });
```

Comment:
You can remove this whole `beforeEach` function - this code is only needed if you're using redux state, because redux uses a big giant global state, which is a pain for testing. With react-query, state is more isolated and you can skip all this mocking and boilerplate. I confirmed that this test suite passes without it.